docs/results.rst

```diff
@@ -19,16 +19,30 @@ configurations for this result are:
 
 -  filename: path to the file. Depending on the file extension this file
    will be saved as excel (xls or xlsx), tsv or csv.
+-  encoding: Defaults to 'utf-8'.
+-  index: Write index. True by default.
+-  header: Write column names. True by default.
+-  float_format: example_.
+-  variables: Dict with variables to be replaced on filename.
+-  extra_args: Extra arguments for to_csv_ or to_excel_ not covered above.
 
 Example of a file result:
 
 .. code:: json
 
     "result": {
       "type": "file",
-      "filename": "output.csv"
+      "filename": "output_{yesterday}.csv",
+      "variables": {"yesterday": "d-1d"},
+      "index": false,
+      "header": false,
+      "extra_args": {"sheet_name": "Very Important Sheet", "start_row": 3}
```

Comment:
Would use arguments that can actually apply to csv (as the example uses csv). I.e. sep, quotechar, etc.